frontend/public/components/namespace.jsx

```diff
@@ -275,6 +275,11 @@ class NamespaceDropdown_ extends React.Component {
         defaultBookmarks={defaultBookmarks}
         storageKey={NAMESPACE_LOCAL_STORAGE_KEY}
         shortCut="n" />
+      <div className="co-create-from-yaml co-m-nav-link">
```

Comment:
This isn't really a nav-link, so the class name is not really accurate.

I'd suggest `co-import-yaml-link` as well.